resources/known-amendments.md

```diff
@@ -857,6 +897,23 @@ Fixes two bugs relating to the handling of NFT transfer fees and trust lines:
 - Adjusts a check for the presence of a trust line when the minter of the NFT is also the issuer of the fungible token that would be paid as a transfer fee. Without this amendment, the [NFTokenCreateOffer transaction][] fails with the result code `tecNO_LINE` if the NFT in question has a transfer fee, the offer amount is denominated in fungible tokens issued by the minter, and the account placing the offer does not have a trust line for those tokens. With the amendment, the offer can be created successfully. (For more detail, see [issue #4941](https://github.com/XRPLF/rippled/issues/4941).)
 
 
+### fixEnforceNFTokenTrustlineV2
+[fixEnforceNFTokenTrustlineV2]: #fixenforcenftokentrustlinev2
+
+| Amendment    | fixEnforceNFTokenTrustlineV2 |
+|:-------------|:-----------------------------|
+| Amendment ID | B32752F7DCC41FB86534118FC4EEC8F56E7BD0A7DB60FD73F93F257233C08E3A |
+| Status       | In Development |
+| Default Vote (Latest stable release) | No |
+| Pre-amendment functionality retired? | No |
+
+Fix a bug where NFT transfer fees could bypass certain limitations on receiving tokens, specifically:
+
+- Prevent an NFT issuer from receiving fungible tokens as transfer fees if the fungible tokens' issuer uses Authorized Trust Lines and the NFT issuer's trust line is not authorized.
```

Comment:
authorized trust lines

Reply:
Changed to lower-case with a link back to the concept article.